pkg/apis/eks.cattle.io/v1/types.go

```diff
@@ -43,6 +43,7 @@ type EKSClusterConfigSpec struct {
 	KmsKey                 *string           `json:"kmsKey" norman:"noupdate,pointer"`
 	PublicAccess           *bool             `json:"publicAccess"`
 	PrivateAccess          *bool             `json:"privateAccess"`
+	EBSCSIDriver           *bool             `json:"ebsCSIDriver"`
```

Comment:
As we are adding it on the CRD spec and making optional (i.e. not auto installing for 1.23 or higher clusters) we'll need a UI change to expose this option.

Reply:
I've created rancher/dashboard#9043 to request this UI change.